features/authoring-ideas.feature

```diff
@@ -0,0 +1,30 @@
+Feature: Authoring ideas
+  In order to share knowledge and initiate legal updates
+  As a citizen
+  I need to be able to author and publish ideas
+
+  Background:
+    Given I am the citizen named Richard S.
+      And there is a tag titled Transparency
+      And there is a tag titled Sovereignty
+
+
+  Scenario: Propose an idea successfully
+    Given there should not be an idea titled "Require libre software in all governmental endeavours."
+      And I am on the homepage
+     When I trigger the floating action button
+        # Ideally, we should wait for the input field instead of waiting for a fixed duration
+#     Then I should be focused on the input field of a new idea (?)
+      And I wait for 1 second
+      And I type "Require libre software in all governmental endeavours."
+        # Here, if we do not wait, an idea without title is submitted.
+        # Not sure what this issue is about nor how to solve it. Help?
+      And I wait for 1 second
```

Comment:
This is really hacky and will probably cause trouble on CI, as `wait` instructions may not be long enough there. It can be fixed (probably), but needs help.

Reply:
Yes, this is hacky. Are you testing that input fields are focusing correctly? I'll test this PR after the rebase.

If I understand well this feature, maybe you can use [hasFocus](http://webdriver.io/api/state/hasFocus.html) instead, and a feature like `And the title get the focus`.

I didn't test if something is focused previously. I usually just type in some field and use 'waitForVisible' and/or 'waitForText'. You can see a sample in [this other meteor app](https://github.com/comunes/todos-contra-el-fuego-web/blob/a6bd1095bf79e1f376f9b27abfbb2fd6ecdf03ea/cucumber/features/steps_definitions/login.js#L35) or login/register in the app.

Reply:
Thanks for the feedback and the useful link ! :heart_eyes: This one is not about focus, nor about waiting for an element (though we can perhaps solve this by waiting for the right DOM element). It seems related to some meteor voodoo, possibly also because we're using `contenteditable` instead of `input`. We're still trudging through the changes since the last rebase, but it's coming along.